Add TPCH Load benchmark

PingLiuPing opened the pull request:
loading (insert) benchmark is missing in pbench, this PR add the initial files for loading benchmark. It includes test files for hive and iceberg connector, both native and Java.

PingLiuPing commented on benchmarks/tpch-load/create-table/customer.sql:
```diff
@@ -0,0 +1,11 @@
+    acctbal     DECIMAL(12,2) NOT NULL,
+    mktsegment  VARCHAR(10) NOT NULL,
+    comment     VARCHAR(117) NOT NULL
+) WITH (format = 'PARQUET');
```
Thank you, just added newline to the end of each file.

ethanyzhang commented on benchmarks/tpch-load/cleanup_sf100.json:
```diff
@@ -0,0 +1,18 @@
+    "drop-table/supplier.sql",
+    "schema/drop_sf100.sql"
+  ],
+  "expected_row_counts": {
```
you can omit this field if you don't have row counts to add

PingLiuPing replied:
Thank you @ethanyzhang .
Just removed this field.

PingLiuPing commented:
@wanglinsong @ethanyzhang  Sorry for the late response, this PR slipped from my mind. I addressed your comments, can you please take another look? Thanks.

PingLiuPing commented:
Hi @wanglinsong  Thanks for your comments, do you think this PR is ready to be merged? Anything else you want me to change?

PingLiuPing commented:
> Have we tested this PR for pbench? @PingLiuPing

Thanks, I didn't test this after fix the review comments. Before that I have tested it in pbench.
So think this commit might not correct https://github.com/prestodb/pbench/pull/55/commits/aee3644c6223a696652547b3f3b9eb03a71cac07.
I will verify this.
